arrow-flight/tests/flight_sql_client_cli.rs

```diff
@@ -245,7 +245,7 @@ impl FlightSqlService for FlightSqlServiceImpl {
             "part_2" => batch.slice(2, 1),
             ticket => panic!("Invalid ticket: {ticket:?}"),
         };
-        let schema = batch.schema();
+        let schema = batch.schema().clone();
         let batches = vec![batch];
         let flight_data = batches_to_flight_data(schema.as_ref(), batches)
```

Comment:
```suggestion
        let schema = batch.schema();
        let batches = vec![batch];
        let flight_data = batches_to_flight_data(schema.as_ref(), batches)
```

Reply:
I tried this change on my machine, but I got a borrow checker error. How does this look instead?
```suggestion
        let schema = batch.schema();
        let batches = vec![batch.clone()];
        let flight_data = batches_to_flight_data(schema, batches)
```